### PR TITLE
Fix documentation to represent ISP being deprecated

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

Fix documentation to represent ISP being deprecated in https://github.com/vitessio/vitess/pull/11557

### CODE REVIEW

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 1]
```diff
@@ -3,36 +3,15 @@ title: Initialize Shard Primary
...
-The command for `InitShardPrimary` is as follows:
-
-```text
-vtctldclient \
-  InitShardPrimary \
-  --force \
-  commerce/0 \
-  cell1-100
-```
+The [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) command should be used to initialize the shard by electing a new primary and setting up replication for the replicas. Additionally, a database is created to store the data for the keyspace-shard. The default name for the database will be `vt_` followed by the keyspace. In our use case, it would be `vt_commerce`. You can override this default by providing an `init_db_name_override` flag to each vttablet. However, all future invocations must continue to supply this parameter.
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: We should keep the instructions for ISP in this page. Even though it is deprecated.
Any details on PRS belong in the PRS page and not here.

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 2]
```diff
@@ -3,36 +3,15 @@ title: Initialize Shard Primary
 weight: 9
 ---
 
-The [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) command should be used to initialize the shard by electing a new primary and setting up replication for the replicas. Additionally, a database is created to store the data for the keyspace-shard. The default name for the database will be `vt_` followed by the keyspace. In our use case, it would be `vt_commerce`. You can override this default by providing an `init_db_name_override` flag to each vttablet. However, all future invocations must continue to supply this parameter.
-
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: ✅

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 3]
```diff
@@ -3,36 +3,15 @@ title: Initialize Shard Primary
...
-The InitShardPrimary step can also be used to do the same operation. However, it is a destructive command and should only be used by advanced users. This command copies over the `executed_gtid_set` from the primary to the replica which can break replication if the user isn't careful. 
-
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md:9]: Let's restore this line, but put it after the new line that says VTOrc will do this automatically.

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 4]
```diff
@@ -3,36 +3,15 @@ title: Initialize Shard Primary
...
 
 ![healthy-tablet](../img/healthy-tablet.png)
 
-{{< warning >}}
-`InitShardPrimary` is a destructive command that resets all servers by deleting their binlog metadata. It should only be used for initializing a brand new cluster.
-{{< /warning >}}
-
-{{< info >}}
-`InitShardPrimary` will soon be deprecated. This action will be performed automatically by VTOrc once it is released as production-ready.
-{{< /info >}}
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Let's restore this, but say that is has already been deprecated and edit the language about VTOrc to remove the "once it is released ..."

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 5]
```diff
@@ -3,36 +3,15 @@ title: Initialize Shard Primary
 weight: 9
 ---
 
-The [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) command should be used to initialize the shard by electing a new primary and setting up replication for the replicas. Additionally, a database is created to store the data for the keyspace-shard. The default name for the database will be `vt_` followed by the keyspace. In our use case, it would be `vt_commerce`. You can override this default by providing an `init_db_name_override` flag to each vttablet. However, all future invocations must continue to supply this parameter.
-
-The InitShardPrimary step can also be used to do the same operation. However, it is a destructive command and should only be used by advanced users. This command copies over the `executed_gtid_set` from the primary to the replica which can break replication if the user isn't careful. 
-
-{{< info >}}
-If using a custom `init_db.sql` that omits `SET sql_log_bin = 0`, then InitShardPrimary must be used instead of PlannedReparentShard.
-{{< /info >}}
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md:12]: ✅

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 3 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md:9]: Fixed.

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 4 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 6]
```diff
@@ -3,13 +3,9 @@ title: Initialize Shard Primary
 weight: 9
 ---
 
-The [PlannedReparentShard](../../configuration-advanced/reparenting/#plannedreparentshard-planned-reparenting) command should be used to initialize the shard by electing a new primary and setting up replication for the replicas. Additionally, a database is created to store the data for the keyspace-shard. The default name for the database will be `vt_` followed by the keyspace. In our use case, it would be `vt_commerce`. You can override this default by providing an `init_db_name_override` flag to each vttablet. However, all future invocations must continue to supply this parameter.
+A new primary is elected automatically by VTOrc and no user action is required. However, a shard can be initialized manually as well using the following steps.
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: If I read this section and immediately read the next paragraph, it doesn't flow smoothly.
"a shard can be initialized manually as well using the following steps.
The InitShardPrimary command can be used to do the same operation."

That is repetitive.

Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]:
```suggestion
A new primary is elected automatically by VTOrc and no user action is required. 
```

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 7]
```diff
@@ -3,13 +3,9 @@ title: Initialize Shard Primary
...
 
-The InitShardPrimary step can also be used to do the same operation. However, it is a destructive command and should only be used by advanced users. This command copies over the `executed_gtid_set` from the primary to the replica which can break replication if the user isn't careful. 
-
-{{< info >}}
-If using a custom `init_db.sql` that omits `SET sql_log_bin = 0`, then InitShardPrimary must be used instead of PlannedReparentShard.
-{{< /info >}}
+The InitShardPrimary command can be used to do the same operation. However, it is a destructive command and should only be used by advanced users. This command copies over the `executed_gtid_set` from the primary to the replica which can break replication if the user isn't careful.
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]:
```suggestion
The InitShardPrimary command can be used to do the same operation manually. However, it is a destructive command and should only be used by advanced users. This command copies over the `executed_gtid_set` from the primary to the replica which can break replication if the user isn't careful.
```

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md [hunk 8]
```diff
@@ -34,5 +30,5 @@ After this step, visiting the `/debug/status` page on the vttablets should show
 {{< /warning >}}
 
 {{< info >}}
-`InitShardPrimary` will soon be deprecated. This action will be performed automatically by VTOrc once it is released as production-ready.
+`InitShardPrimary` is deprecated. This action is performed automatically by VTOrc.
```
Comment (deepthi) [inline content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]:
```suggestion
`InitShardPrimary` is deprecated. This action is performed automatically by VTOrc. If manual action is needed, it is recommended to use `PlannedReparentShard`.
```

**Review by deepthi — APPROVED** [review]
Approved with feedback. Can merge once it is all addressed.

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 6 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 7 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

File: content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md (hunk 8 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/configuration-basic/initialize-shard-primary.md]: Fixed

**Pull request merged**